main.js

```diff
@@ -0,0 +1,147 @@
+var VideoModel = Backbone.Model.extend({
+  defaults: {
+    videoId: '',
+    title: '',
+    description: '',
+    thumbnails: '',
+  },
+});
+
+var AppModel = Backbone.Model.extend({
+  defaults: function () {
+    return {
+      videos: new VideosCollection(),
+      searchTerm: '',
+      currentVideo: null,
+      apiKey: 'AIzaSyCXJ4dabPUSUUVCbFRMWJP59dgxIU28pOM',
+    };
+  },
+
+  initialize: function () {
+    //default search
+    this.set('searchTerm', 'babies laughing at dogs'); 
+    this.updateSearch(); 
+  }, 
+
+  updateSearch: function () {
+    this.get('videos').makeFetch(this.get('searchTerm'), this.get('apiKey'));
+  },
+  
+  updateCurrentVideo: function (videoId) {
+    var allVideos = this.get('videos');
+    var currentVideo = allVideos.findWhere({videoId: videoId});
+
+    this.set('currentVideo', currentVideo);
+  },
+});
+
+var VideosCollection = Backbone.Collection.extend({
+  url:  '',
+
+  model: VideoModel,
+
+  makeFetch: function (searchTerm, apiKey) {
```

Comment:
Hmm - just opinion, but this is just a strange name for a function ha - perhaps just call it `fetchVideos`?